bcda/cclf/testutils/cclfUtils.go

```diff
@@ -101,26 +105,32 @@ func ImportCCLFPackage(acoSize, environment string) (err error) {
 	}
 }
 
-func AddFileToZip(zipWriter *zip.Writer, filename string) error {
-	sourceData := strings.Split(filename,"__")
+// AddFileToZip adds the file to a zip archive
+func AddFileToZip(zipWriter *zip.Writer, filename string) (err error) {
+	sourceData := strings.Split(filename, "__")
 	src := sourceData[0]
 	filename = sourceData[1]
 
 	fileToZip, err := os.Open(filepath.Clean(src))
 	if err != nil {
-		return err
+		return
```

Comment:
Is there a reason for removing the `err` from the `return` statement?

Reply:
I removed `err` because it wasn't strictly necessary after modifying the function signature to use a named return value. Using a named return value allows me to set the return value in the closure method that is deferred. This is testing my golang knowledge. Can you return a value from a function that is being called in a defer statement?

Reply:
In this particular case, the `err` that is being returned is related to the call to `os.Open()`, which is completely separate from the usage of `defer`.  So, by removing the `err` from the return statement, we'd lose the error reason if the file fails to open.  I think this can be considered a global comment.

Reply:
Oops. Thanks for bearing with me as I get familiar with golang :smile: 



Thanks for bearing with me as I get myself familiarized with golang :smile: